src/sentry/south_migrations/0342_auto__add_index_groupcommitresolution_commit_id.py

```diff
@@ -0,0 +1,3245 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+    def forwards(self, orm):
+        # Adding index on 'GroupCommitResolution', fields ['commit_id']
+        db.create_index('sentry_groupcommitresolution', ['commit_id'])
```

Comment:
so matt can correct me if this is unnecessary in this case, but generally we want to create indices concurrently for postgres. if you search for `CREATE INDEX CONCURRENTLY` there should be some examples where we've done that in the past for migrations